src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -59,14 +59,14 @@ public override BinaryAnalyzerContext InitializeGlobalContextFromOptions(Analyze
                 context.Logger = aggregatingLogger;
             }
 
-            base.InitializeGlobalContextFromOptions(options, ref context);
-
             // We override the driver framework size default to be as large as
             // possible Binaries and (in particular) their PDBs can be large.
             context.MaxFileSizeInKilobytes = options.MaxFileSizeInKilobytes != null
                 ? options.MaxFileSizeInKilobytes.Value
                 : long.MaxValue;
```

Comment:
This code was too late, the files were already skipped

Reply:
Thanks for this fix!